spec/models/expense_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
```

Comment:
- [ ] Good job working on the changes suggested by the previous reviewers. However, kindly work on the tests because most of the tests failed. I checkout to your previous commits before you submitted them for first and second reviews and they all passed at that time but then they fail after you made changes suggested by the previous reviewers. Kindly fix this so that your tests will pass as before.

Before the previous reviews | After you worked on the review changes
:---:|:---:
![](https://i.imgur.com/pFJTs9X.png) | ![](https://i.imgur.com/upEaXIT.png)